bot/modules/claimbot.js

```diff
@@ -0,0 +1,325 @@
+"use strict";
+
+let lbry;
+let mongo;
+let discordBot;
+let moment = require("moment");
+let request = require("request");
+let sleep = require("sleep");
+let config = require("config");
+let channels = config.get("claimbot").channels;
+const Discord = require("discord.js");
+
+module.exports = {
+  init: init
+};
+
+function init(discordBot_, mongodburl) {
```

Comment:
this should be a exported function, example follows in the example files for a plugin https://github.com/lbryio/lbry-wunderbot/blob/master/bot/modules/example.js

Reply:
discordBot will automatically be the discordBot when it´s imported so no need to change it, mongodb url should be requested from config file i.e the only var needed here would be discordBot_ :)